docs/src/docs/usage/install/index.mdx

```diff
@@ -53,7 +53,13 @@ brew install golangci-lint
 brew upgrade golangci-lint
 ```
 
-Note: previously we use [tap](https://github.com/golangci/homebrew-tap), but it is obsoleted. We recommend using official formula instead of an obsolete tap.
+Note: Previously we used a [homebrew tap](https://github.com/golangci/homebrew-tap). We recommend using official formula instead of the tap, but sometimes the most recent release 
+isn't immediately available via homebrew core. In this case, the tap formula can be used to install the latest version of `golangci-lint`:
```

Comment:
Could you please add a bit more details, like the tap formula is updated automatically, when the core requires some manual actions and can be delivered with a delay

Reply:
Updated, but let me know if I can be more specific with anything.